packages/@glimmer/compiler/lib/javascript-compiler.ts

```diff
@@ -182,6 +236,10 @@ export default class JavaScriptCompiler<T extends TemplateMeta> {
     this.template.block.yields.add(to);
   }
 
+  debugger() {
+    this.push(['debugger', null, null]);
+  }
```

Comment:
@chadhietala what is the reason behind pushing this into the pre-compiler, vs just specializing it at runtime?